spec.html

```diff
@@ -43695,6 +43745,10 @@ <h1>
         <emu-alg>
           1. Let _elementSize_ be the Element Size value specified in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for Element Type _type_.
           1. If _isLittleEndian_ is *false*, reverse the order of the elements of _rawBytes_.
+          1. If _type_ is ~float16~, then
+            1. Let _value_ be the byte elements of _rawBytes_ concatenated and interpreted as a little-endian bit string encoding of an IEEE 754-2019 binary16 value.
+            1. If _value_ is an IEEE 754-2019 binary16 NaN value, return the *NaN* Number value.
```

Comment:
```suggestion
            1. If _value_ is an IEEE 754-2019 binary16 format Not-a-Number encoding, return the *NaN* Number value.
```

for consistency with phasing in `NumericToRawBytes`. Specifically, I'd prefer not to use "NaN" to refer to anything other than a binary64 NaN.

Reply:
We could also just make a quick PR addressing these editorial concerns and rebase your proposal PR on top of it.

Reply:
I don't think it makes sense to care about rushing the editorial change in before landing this PR given that this exact wording has stood for most of a decade. I'd prefer to land this in a form consistent with the existing spec and then bikeshed any other editorial changes to the surrounding stuff at our leisure.

Reply:
Agree with @bakkot, consistency for now, debating wording changes after.